kfs-core/src/main/java/edu/arizona/kfs/pdp/businessobject/ShippingAccount.java

```diff
@@ -0,0 +1,209 @@
+	private String shippingAccountNumber;
+	private String shippingAccountType;
+	private String accountNumber;
+	private String financialObjectCode;
+	private String chartOfAccountsCode;
+	private String contactPrincipalId;
+	private String subAccountNumber;
+	private String financialSubObjectCode;
+	private String projCode;
+	private String organizationReferenceId;
+	private boolean active;
+	
+	private Person contactUser;
+	private Account account;
+	private Chart chartOfAccounts;
+	private SubAccount subAccount;
+	private ObjectCode	objectCode;
+	private SubObjectCode subObjectCode;
+	private ProjectCode	projectCode;
+	private Person accountFiscalOfficerUser;
+	
+	public String getShippingAccountNumber() {
+		return shippingAccountNumber;
+	}
+	
+	public void setShippingAccountNumber(String shippingAccountNumber) {
+		this.shippingAccountNumber = shippingAccountNumber;
+	}
+	
+	public String getShippingAccountType() {
+		return shippingAccountType;
+	}
+	
+	public void setShippingAccountType(String shippingAccountType) {
+		this.shippingAccountType = shippingAccountType;
+	}
+	
+	public String getAccountNumber() {
+		return accountNumber;
+	}
+	
+	public void setAccountNumber(String accountNumber) {
+		this.accountNumber = accountNumber;
+	}
+	
+	public String getFinancialObjectCode() {
+		return financialObjectCode;
+	}
+	
+	public void setFinancialObjectCode(String financialObjectCode) {
+		this.financialObjectCode = financialObjectCode;
+	}
+	
+	public String getChartOfAccountsCode() {
+		return chartOfAccountsCode;
+	}
+	
+	public void setChartOfAccountsCode(String chartOfAccountsCode) {
+		this.chartOfAccountsCode = chartOfAccountsCode;
+	}
+	
+	public String getContactPrincipalId() {
+		return contactPrincipalId;
+	}
+	
+	public void setContactPrincipalId(String contactPrincipalId) {
+		this.contactPrincipalId = contactPrincipalId;
+	}
+	
+	public String getSubAccountNumber() {
+		return subAccountNumber;
+	}
+	
+	public void setSubAccountNumber(String subAccountNumber) {
+		this.subAccountNumber = subAccountNumber;
+	}
+	
+	public String getFinancialSubObjectCode() {
+		return financialSubObjectCode;
+	}
+	
+	public void setFinancialSubObjectCode(String financialSubObjectCode) {
+		this.financialSubObjectCode = financialSubObjectCode;
+	}
+	
+	public String getProjCode() {
+		return projCode;
+	}
+	
+	public void setProjCode(String projCode) {
+		this.projCode = projCode;
+	}
+	
+	public String getOrganizationReferenceId() {
+		return organizationReferenceId;
+	}
+	
+	public void setOrganizationReferenceId(String organizationReferenceId) {
+		this.organizationReferenceId = organizationReferenceId;
+	}
+	
+	public boolean isActive() {
+		return active;
+	}
+	
+	public void setActive(boolean active) {
+		this.active = active;
+	}
+	
+	public Person getContactUser() {
+		contactUser = SpringContext.getBean(org.kuali.rice.kim.api.identity.PersonService.class).updatePersonIfNecessary(contactPrincipalId, contactUser);
```

Comment:
Could you just use PersonService.class here and add the import instead of listing the fully qualified name?